app/serializers/player_serializer.py

```diff
@@ -5,6 +5,33 @@
 from app.models.player import PlayerRole
 from app.serializers import FactionSerializer, GameSerializer
 
+class SimplePlayerSerializer(serializers.ModelSerializer):
+    roleChar = serializers.SerializerMethodField('getRole')
+
+    def getRole(self, player):
+        return player.role.value
+
+    name = serializers.SerializerMethodField('getName')
+
+    def getName(self, player):
+        return player.user.get_full_name()
+
+    processedScore = serializers.SerializerMethodField('getProcessedScore')
+
+    def getProcessedScore(self, player):
+        if player.is_score_public:
+            return player.score()
+        else:
+            return -1
+    
+    class Meta:
+        model = Player
+        fields = ['roleChar', 
+            'name',
+            'processedScore']
+
+    point_modifier: int = models.IntegerField(default=0)
```

Comment:
What does this line achieve and why do we need it? I may be missing something here.

Reply:
My bad. Removing it now.